src/components/QuestionEdit/index.tsx

```diff
@@ -41,6 +41,16 @@ export default function QuestionEdit({
   const { showAlert } = useStore();
   const options = watch('options');
   const isWithoutOptions = options.length === 1;
+  const { points, text } = watch();
```

Comment:
Maybe take extract options here. No need of line 42
```suggestion
  const { points, text, options } = watch();
  const isWithoutOptions = options.length === 1;
```